### PR TITLE
added both group types

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/enterprise/mam-policies-for-your-microsoft-365-enterprise-dev-test-environment.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ In this phase, you create a device compliance policy for Windows 10 devices.
     
 6. On the **Groups-All groups** blade, click **+ New Group**.
     
-7. On the **Group** blade, select **Office 365** for **Group type?**, type **Managed Windows 10 device users** in **Name**, select **Assigned** in **Membership type**,  and then click **Create**. 
+7. On the **Group** blade, select **Office 365** or **Security** for **Group type?**, type **Managed Windows 10 device users** in **Name**, select **Assigned** in **Membership type**,  and then click **Create**.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/mam-policies-for-your-microsoft-365-enterprise-dev-test-environment.md:52]:
```suggestion
7. On the **Group** blade, select **Office 365** or **Security** for **Group type?**, type in **Managed Windows 10 device users** in the**Name** field, select **Assigned** in **Membership type**,  and then click **Create**. 
```

File: microsoft-365/enterprise/mam-policies-for-your-microsoft-365-enterprise-dev-test-environment.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ In this phase, you create a device compliance policy for Windows 10 devices.
 
 26. On the **Add app** blade, click **Add**.
 
-You now have a device compliance policy for testing the selected apps in the **Windows 10** device compliance policy and for members of the **Managed Windows 10 device users** group. 
+You now have a device compliance policy for testing the selected apps in the **Windows 10** device compliance policy and for members of the **Managed Windows 10 device users** group. Please make note selecting Office 365 as the group type will create additional resources.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/mam-policies-for-your-microsoft-365-enterprise-dev-test-environment.md]:
```suggestion
You now have a device compliance policy for testing the selected apps in the **Windows 10** device compliance policy and for members of the **Managed Windows 10 device users** group. Please note that selecting Office 365 as the group type will create additional resources. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - These are only suggestions to improve readability but I'm not sure if it will impact on technical integrity.

**Comment by mypil** [comment]
@Malind19 - Kindly review on the suggested changes made by JohanFreelancer9 and make the necessary changes on the PR. Thank you.

**Comment by Malind19** [comment]
@mypil Done. Thanks.

**Comment by mypil** [comment]
@JoeDavies-MSFT - Can you please assist to check, approve and merge the content for this PR based on issue #339?

Thank you.

**Comment by JoeDavies-MSFT** [comment]
Done.

**Pull request merged**